01_prepare_host.sh

```diff
@@ -164,14 +137,16 @@ remove_ironic_containers
 
 # Clean-up existing pod, if podman
 case "${CONTAINER_RUNTIME}" in
-podman)
-  for pod in ironic-pod infra-pod; do
-    if  sudo "${CONTAINER_RUNTIME}" pod exists "${pod}" ; then
-        sudo "${CONTAINER_RUNTIME}" pod rm "${pod}" -f
-    fi
-    sudo "${CONTAINER_RUNTIME}" pod create -n "${pod}"
-  done
-  ;;
+  podman)
+    for pod in ironic-pod infra-pod; do
+      if  sudo "${CONTAINER_RUNTIME}" pod exists "${pod}" ; then
+          sudo "${CONTAINER_RUNTIME}" pod rm "${pod}" -f
+      fi
+      sudo "${CONTAINER_RUNTIME}" pod create -n "${pod}"
+    done
+    ;;
+  *)
```

Comment:
what is this line doing?

Reply:
It is fallback option for `case`, mandatory per shellcheck even if its just a fallthru like this one. We should error out if someone specifies `CONTAINER_RUNTIME` we don't support, but that is for another PR.